mlflow/tensorflow.py

```diff
@@ -758,32 +758,6 @@ def fit(self, *args, **kwargs):
             _flush_queue()
             _log_artifacts_with_warning(local_dir=log_dir, artifact_path='tensorboard_logs')
             shutil.rmtree(log_dir)
-
-            return result
-
-    @gorilla.patch(tensorflow.keras.Model)
-    def fit_generator(self, *args, **kwargs):
```

Comment:
If `fit_generator()` is only deprecated in TF 2.1.0, do we need to remove it from TF 1.X? Are we trying to keep both versions consistent with each other?

Reply:
Good question, we only have a single autologging patch for `tf.keras.fit_generator`, so removing `fit_generator` support here removes it for both 1.x & 2.x. It seems 1.x users should also be able to pass a generator to `fit`, as per the TF 1.14 API docs: https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/keras/Model#fit

Reply:
Ah I had thought that if `fit_generator` did something different from `fit`, we could've patched the method for TF 1.X only by having the version check before the patch. But it turns out that `fit` [just calls](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/engine/training.py#L682) `fit_generator()`. LGTM!